.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+1. If this is your first time, please read our contributor guidelines: https://github.com/m3db/m3/blob/master/CONTRIBUTING.md and developer notes https://github.com/m3db/m3/blob/master/DEVELOPER.md
+2. Please label this pull request according to what type of issue you are addressing.
```

Comment:
also maybe add: e.g. if this is a pr for the query service add `[query]` to the title of your pr

Reply:
Sure, it will be hard to enforce... but I can add it.